include/cantera/oneD/Boundary1D.h

```diff
@@ -106,6 +106,10 @@ class Inlet1D : public Boundary1D
 public:
     Inlet1D();
 
+    Inlet1D(shared_ptr<Solution> sol) : Inlet1D(){
```

Comment:
```suggestion
    Inlet1D(shared_ptr<Solution> solution) : Inlet1D(){
```
And likewise elsewhere